### ISSUE TITLE
published version on crates.io has executable attribute on all files

### ISSUE THREAD

**Issue by ignatenkobrain** [kvark/copyless#12]
```
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/bors.toml is executable but has empty or no shebang, removing executable bit
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/LICENSE is executable but has empty or no shebang, removing executable bit
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/src/boxed.rs is executable but has empty or no shebang, removing executable bit
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/src/vec.rs is executable but has empty or no shebang, removing executable bit
*** ERROR: ./usr/share/cargo/registry/copyless-0.1.2/src/lib.rs has shebang which doesn't start with '/' ([warn(missing_docs)])
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/rustfmt.toml is executable but has empty or no shebang, removing executable bit
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/CHANGELOG.md is executable but has empty or no shebang, removing executable bit
*** WARNING: ./usr/share/cargo/registry/copyless-0.1.2/README.md is executable but has empty or no shebang, removing executable bit
```

That is creating problems when packaging for Fedora.

**Comment by kvark** [comment]
I was publishing it from a shared NTFS driver between Linux and Windows. Guess sharing this way comes with disadvantages...

**Comment by kvark** [comment]
Fixed in 301fae567f518bab31a136461bc8dcba955a87f6
